If the seed resource is missing the secretRef block to point to the seed access credentials, gardenctl panic errors when trying to target seed

**Describe the bug**
When creating an external seed, the instructions did not specify to include the block secretRef in the spec for the seed or create the appropiate seed secret. This does not create any issues with the operation of the seed, however it impacts the ability of to use gardenctl to target the seed.  Gardenctl does not properly handle the error of the missing secret reference or the missing seed and should be updated to properly detect the condition.

Overall the condition I believe should not exist when creating external seeds, but regardless gardenctl should be able to properly handle the condition and report the issue.

The expected format of the spec section of the seed resource  is below, to recreate the issue, remove secretRef, and then try to target the designated seed.
"spec": {
			"backup": {
				...
				}
			},
			"dns": {
				...
			},
			"networks": {
				...
			},
			"provider": {
				...
			},
			"secretRef": {
				"name": "seed-us-gov-west-1",
				"namespace": "garden"
			},
			"settings": {
				...
			}
		}

**To Reproduce**
Steps to reproduce the behavior:
Within a seed resource, edit to remove the spec.secretRef block

1. Which target was set 'gardenctl get target'. gardenctl get target -g <garden> -s <seed>
2. Which command was entered [e.g. 'gardenctl show vpn-seed'] gardenctl get target -g <garden> -s <seed>
3. What was the output of the command 

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1e4dd46]

goroutine 1 [running]:
github.com/gardener/gardenctl/pkg/cmd.targetSeed(0x22f2260, 0x2ddf120, 0x22f2280, 0x2ddf120, 0xc0002ef240, 0x1a, 0x1)
	/Users/d068290/go/src/github.com/gardener/gardenctl/pkg/cmd/target.go:452 +0x1f6
github.com/gardener/gardenctl/pkg/cmd.seedWrapper(0x22f2260, 0x2ddf120, 0x22f2280, 0x2ddf120, 0x22f2200, 0x2ddf120, 0x22f3760, 0xc0000be000, 0x22f3780, 0xc0000be008, ...)
	/Users/d068290/go/src/github.com/gardener/gardenctl/pkg/cmd/target.go:989 +0x29c
github.com/gardener/gardenctl/pkg/cmd.NewTargetCmd.func1(0xc0002e6000, 0xc000263300, 0x0, 0x4, 0x0, 0x0)
	/Users/d068290/go/src/github.com/gardener/gardenctl/pkg/cmd/target.go:86 +0x5da
github.com/spf13/cobra.(*Command).execute(0xc0002e6000, 0xc000263240, 0x4, 0x4, 0xc0002e6000, 0xc000263240)
	/Users/d068290/go/src/github.com/gardener/gardenctl/vendor/github.com/spf13/cobra/command.go:826 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x2da4140, 0x1d, 0xc0000bdc40, 0x36)
	/Users/d068290/go/src/github.com/gardener/gardenctl/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/d068290/go/src/github.com/gardener/gardenctl/vendor/github.com/spf13/cobra/command.go:864
github.com/gardener/gardenctl/pkg/cmd.Execute()
	/Users/d068290/go/src/github.com/gardener/gardenctl/pkg/cmd/root.go:111 +0x3aa
main.main()
	/Users/d068290/go/src/github.com/gardener/gardenctl/cmd/gardenctl/main.go:20 +0x20

**Expected behavior**
gardenctl should report an error that it is unable to target seed due to missing information.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Gardenctl Version (please complete the following information): v0.19.0 and v0.21.0
 - Version/Commit [e.g. b21b4bde14663faee381697cb1d93a8b53a3e81a]

**Additional context**
Add any other context about the problem here.

--- a/pkg/cmd/target.go
+++ b/pkg/cmd/target.go
@@ -450,6 +450,10 @@ func targetSeed(targetReader TargetReader, targetWriter TargetWriter, name strin
 		fmt.Println("Seed not found")
 		os.Exit(2)
 	}
+	if seed.Spec.SecretRef == nil {
+		fmt.Println("Spec.SecretRef is missing in this seed, seed not reachable")
+		os.Exit(2)
+	}
 	kubeSecret, err := Client.CoreV1().Secrets(seed.Spec.SecretRef.Namespace).Get(seed.Spec.SecretRef.Name, metav1.GetOptions{})
 	checkError(err)
 	pathSeed := filepath.Join(pathGardenHome, "cache", gardenName, "seeds", name)

